website/thunderbird/91.0/whatsnew/index.html

```diff
@@ -0,0 +1,190 @@
+        </p>
+      </aside>
+    </div>
+  </section>
+
+  {% if not LANG.startswith('zh') %}
+  <!-- Mozilla VPN banner -->
+  <section class="pt-20 pb-20 flex justify-center bg-grey-light">
+    <div class="flex items-center lg:flex-row flex-col flex-col-reverse flex-1 max-w-6xl ml-16 mr-16">
+      <aside class="lg:mr-8">
+        <img class="w-full h-auto max-w-md self-center" src="{{ static('/media/img/thunderbird/whatsnew/91/vpn_one_tap.svg') }}" alt="Mozilla VPN vector graphic">
+      </aside>
+      <aside class="flex-col flex-1 items-start">
+        <h3 class="header-section text-green-light">
+          <span class="green">{{ svg('accessibility') }}</span>
+          {{ _('Protect Your Privacy')}}
+        </h3>
+        <p class="font-lg tracking-wider leading-loose mb-10">
+          {{ _('Whether its work or play, you deserve to have privacy when you are online. A VPN, Virtual Private Network, can help you create a secure, private connection to the internet. So that when you are traveling, connecting to WiFi, or simply want more online security - you know that your information is protected from prying eyes.') }}
+        </p>
+        <a href="https://www.mozilla.org/products/vpn/?utm_source=thunderbird&utm_medium=tbwnp&utm_campaign=2021" class="btn-body self-start" target="_blank">
+          {{_('Learn More')}} <span>{{ svg('chevron-right') }}</span>
+        </a>
+      </aside>
+    </div>
+  </section>
+  {% endif %}
+
+  <!-- Calendar sidebar -->
+  <section class="pt-20 pb-20 flex justify-center">
+    <div class="flex lg:flex-row flex-col flex-col-reverse flex-1 max-w-6xl ml-16 mr-16">
+      <aside class="flex-1 lg:mr-8">
+        <img class="w-full h-auto max-w-2xl self-center shadow-img" src="{{ static('/media/img/thunderbird/whatsnew/91/thunderbird-91-calendar-sidebar.gif') }}" alt="Screenshot of the new options in the Thunderbird calendar sidebar.">
+      </aside>
+      <aside class="flex-col flex-1 items-start">
+        <h3 class="header-section">
+          <span>{{ svg('calendar') }}</span>
+          {{ _('Calendar Sidebar')}}
+        </h3>
+        <p class="font-lg tracking-wider leading-loose mb-10">
+          {{ _('Managing multiple calendars has been made easier with the calendar sidebar improvements in this release. There is a quick enable button for disabled calendars, as well as a show/hide icon for easily toggling what calendars are visible. There is also a lock indicator for read-only calendars. Additionally, although not a sidebar improvement, there are now better color accents to highlight the current day in the calendar.') }}
```

Comment:
@ryanleesipes This is so awkward lol, can't this section just be named 'Calendar Improvements' and forego explaining the last bit is not a sidebar improvement?

Reply:
Yeah, I'm okay with changing the name of this section.